js/instant-articles-meta-box.js

```diff
@@ -1,3 +1,13 @@
+function instant_articles_force_submit ( post_ID ) {
+	var data = {
+		'action': 'instant_articles_force_submit',
+		'post_ID': post_ID,
+		'force': jQuery( '#instant_articles_force_submit' ).is( ':checked' )
```

Comment:
As this is an admin action that updates the database we should be setting and [verifying a nonce](https://codex.wordpress.org/Function_Reference/check_ajax_referer).